pkg/osquery/tables/crowdstrike/table.go

```diff
@@ -0,0 +1,68 @@
+package crowdstrike
+
+import (
+	"context"
+	"regexp"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/launcher/pkg/osquery/tables/dataflattentable"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+type Table struct {
+	name   string
+	logger log.Logger
+}
+
+const tableName = "kolide_crowdstrike"
+
+var re = regexp.MustCompile(`\s?=\s?|\s`)
+
+func TablePlugin(logger log.Logger) *table.Plugin {
```

Comment:
If there will be two tables in here, these should be named somewhat more distinctly. 

Also, probably don't need to export the struct.